.gitignore

```diff
@@ -7,4 +7,6 @@ venv
 /game_data.csv
 game_data.csv
 /.game_data.csv
-.game_data.csv
\ No newline at end of file
+.game_data.csv
+
+__pycache__
```

Comment:
`pytest` created the `__pychache__` directory which we don't need in version control.